.github/workflows/pullrequest.yml

```
@@ -51,18 +53,13 @@ jobs:
       LINT_CHUNKS: ${{ steps.lint_projects.outputs.CHUNKS }}
       BUILD_CHUNKS: ${{ steps.build_projects.outputs.CHUNKS }}
       NX_BASE: ${{ steps.export-sha.outputs.NX_BASE }}
+      NX_HEAD: ${{ steps.export-sha.outputs.NX_HEAD }}
       DEPLOY_FEATURE: ${{ contains(github.event.pull_request.labels.*.name, 'deploy-feature') }}
       CACHE_KEY: ${{ steps.set-cache-key.outputs.CACHE_KEY }}
```

actionlint (1.7.4)

property "set-cache-key" is not defined in object type {build_projects: {conclusion: string; outcome: string; outputs: {string => string}}; check-permission: {conclusion: string; outcome: string; outputs: {string => string}}; check-release-manager-approval: {conclusion: string; outcome: string; outputs: {string => string}}; e2e_projects: {conclusion: string; outcome: string; outputs: {string => string}}; export-sha: {conclusion: string; outcome: string; outputs: {string => string}}; lint_projects: {conclusion: string; outcome: string; outputs: {string => string}}; test_projects: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)